input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,295 @@ Example: Patient resource with interpreter required and language is known
 }
 ~~~
 
+### Representing body site and laterality
+When using some AU Core profiles it may be desirable to represent relevant body site and laterality information associated with a record using CodeableConcept elements, notably:
+*  [AU Core Condition](StructureDefinition-au-core-condition.html) - with primary finding code `Condition.code` and `Condition.bodySite`
+*  [AU Core Procedure](StructureDefinition-au-core-procedure.html) - with primary procedure code `Procedure.code` and `Procedure.bodySite`
+
+When recommending approaches to this, considerations include:
+* The `bodySite` element is NOT `Must Support` in AU Core profiles, there is no expectation to fill or meaningfully consume this element.
+* The `CodeableConcept.text` element is system populated and may reflect more specific detail than the `CodeableConcept.coding` concepts provided.
+
+Body site and associated laterality may be recorded in various ways. Depending on the specific system approach to body site / laterality related content this can lead to a number of scenarios needing to be supported with using AU Core profiles:
+
+a) Body site with laterality is defined in a pre-coordinated primary finding/procedure code.
+
+b) Body site without laterality is defined in a primary finding/procedure code AND coded laterality qualifier is separate.
```

Comment:
Suggest a rewrite of these points, some concern is use of word "define" we want more of a verb that is about knowing/including/having/present rather than "define". Possibly doesn't need a verb (something like the below which still needs tweaks):
1. Primary finding/procedure code with body site and laterality in a pre-coordinated code.
2. Primary finding/procedure code with body site (without laterality) in a pre-coordinated code, and a separate coded laterality qualifier.
3. Body site with laterality and separate primary finding/procedure code.
d) Body site without laterality and separate coded laterality qualifier and separate primary finding/procedure code.